packages/nimble-tokens/build/svg-to-ts-config.js

```diff
@@ -10,25 +10,28 @@ const svgToTsConfig = {
     srcFiles: [
         iconAssetDir
     ],
-    conversionType: 'constants',
     prefix: '',
     svgoConfig: {
         // https://github.com/svg/svgo#built-in-plugins
         plugins: [
             {
-                cleanupAttrs: true
-            },
-            {
-                removeUnknownsAndDefaults: {
-                    keepDataAttrs: false
-                }
-            },
+                name: 'preset-default',
+                params: {
+                    overrides: {
+                        // customize default plugin options
+                        removeUnknownsAndDefaults: {
+                            keepDataAttrs: false,
+                        },
+                    },
+                },
+            }
         ],
     },
     fileName: 'index',
     outputDirectory: outputDir,
     interfaceName: 'NimbleIcon',
-    typeName: 'NimbleIconName'
+    typeName: 'NimbleIconName',
+    exportCompleteIconSet: false
```

Comment:
this was a new before that exported an icon named `complete` which bundled all icons into one string, unneeded so disabled.